apis/proto/v1/mirror/mirror.proto

```diff
@@ -0,0 +1,47 @@
+//
+// Copyright (C) 2019-2022 vdaas.org vald team <vald@vdaas.org>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+syntax = "proto3";
+
+package mirror.v1;
+
+import "apis/proto/v1/payload/payload.proto";
+import "github.com/googleapis/googleapis/google/api/annotations.proto";
+
+option go_package = "github.com/vdaas/vald/apis/grpc/v1/mirror";
+option java_multiple_files = true;
+option java_package = "org.vdaas.vald.api.v1.mirror";
+option java_outer_classname = "ValdMirror";
+
+
+// Represent the mirror service.
+service Mirror {
+
+  // Register the RPC to register other mirror servers.
+  rpc Register(payload.v1.Mirror.RegisterRequest) returns (payload.v1.Empty) {
+    option (google.api.http) = {
+      post : "/mirror/register"
+      body : "*"
+    };
+  }
+
+  // Targets the RPC to get other mirror servers.
+  rpc Targets(payload.v1.Mirror.TargetsRequest) returns (payload.v1.Mirror.Targets) {
+    option (google.api.http) = {
+      post : "/mirror/targets"
```

Comment:
get

Reply:
:memo:
Since it was decided in the discussion to accept the ip, I will close this comment.